Classes/Bookmark/BookmarkModel.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  BookmarkModel.swift
+//  Freetime
+//
+//  Created by Rizwan on 18/10/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+final class BookmarkModel: NSObject, NSCoding, Filterable {
```

Comment:
Project is in Xcode 9 now, would we be able to use `Codable` here to be "more swifty"?

Reply:
I'm not too opinionated here. Familiar w/ `NSCoding` and idk the ups or downs of `Codable`.